facegame/common/static/js/fgscript.js

```diff
@@ -165,8 +167,44 @@ $(document).ready(function () {
   $('.switchimg').tipsy();
 
   new_thumbs();
+
+  $(".siteChoice").on("click", function() {
+    $(this).next().find('.glyphicon').toggleClass('active');
+    if ($(':checked').size() == 0) {
```

Comment:
What version of jQuery is this project using? `size()` is deprecated since 1.8 and `.length` should be used instead.